lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -332,6 +313,10 @@ def test_entrance_exam_requirement_message_with_correct_percentage(self):
         minimum_score_pct = 29
         self.course.entrance_exam_minimum_score_pct = float(minimum_score_pct) / 100
```

Comment:
Bah, I guess `entrance_exam_minimum_score_pct` has already set the precedent of storing `percentage / 100` as a percentage. Nevermind about my nits then, I guess that boat has already sailed.